<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgOptionsHeightMap</class>
 <widget class="QWidget" name="WdgOptionsHeightMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>145</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Size:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KisIntParseSpinBox" name="intSize">
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Endianness:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioMac">
          <property name="toolTip">
           <string>Big Endian</string>
          </property>
          <property name="text">
           <string>&amp;Mac</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioPC">
          <property name="toolTip">
           <string>Little Endian</string>
          </property>
          <property name="text">
           <string>PC</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisIntParseSpinBox</class>
   <extends>QSpinBox</extends>
   <header>kis_int_parse_spin_box.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
